content/en/Platform Deep Dive/Pentests/Reports/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Pentest Reports"
+linkTitle: "Reports"
+weight: 10
+description: >
+  Get a summary of vulnarabilities in your software.
+---
+
+{{% pageinfo %}}
+Download a pentest report to view security issues that our pentesters found.
+{{% /pageinfo %}}
+
+If you want a pentest report, you generally must set up a test of at least eight (8) credits. If you’ve set up a pentest with fewer credits, you’ll still have access to the non-report items listed in Pentest Expectations.
```

Comment:
We have a shortcode with numbers of credits in `/layouts/shortcodes/min-credits-report.md`. Maybe:

```suggestion
{{% min-credits-report %}}. If you’ve set up a pentest with fewer credits, you’ll still have access to the non-report items listed in Pentest Expectations.
```